applications/matter_bridge/README.rst

```diff
@@ -156,6 +156,7 @@ Adding a simulated bridged device to the Matter bridge
      The argument is mandatory and accepts the following values:
 
       * ``256`` - On/Off Light.
+      * ``291`` - Environmental Sensor (a composition of Temperature Sensor and Humidity Sensor).
```

Comment:
Shouldn't it be moved below into the `Adding a Bluetooth LE bridged device to the Matter bridge` section?

Reply:
And why would that be? This is a list of device types supported by the `add` command.

Reply:
But we have another section regarding BLE devices with its examples below and the Environmental Sensor is a BLE device, isn't it? Or, if it is supported as BLE and simulated device I think it should be mentioned in both sections.

Reply:
![image](https://github.com/nrfconnect/sdk-nrf/assets/98917584/3f689603-b92b-42fd-b69f-2aea1c8e16da)

Reply:
Ahh ok, now I got you. Will fix it.